### ISSUE TITLE
question: How well does this work with local repos?

### ISSUE THREAD

**Issue by tecosaur** [greenpau/caddy-git#13]
Hello!

I'm currently putting together a personal server VM with Gitea and Caddy. I'm wondering if it might be possible to have my homepage and blog's static sources as a git repo be used by caddy with this plugin? E.g. `/var/lib/gitea/repositories/tec/website.git`. For non-static pages I'm thinking a dedicated "static/generated content" branch could be generated by CI. I'm well aware this would not be a standard approach, but for my situation (and maybe others?), it seems like this could be a convenient way of setting things up.

Does this use case fit well with this plugin?

**Comment by greenpau** [comment]
> Does this use case fit well with this plugin?

@tecosaur , yes. I could see how it could be useful. I never tried the plugin with local repos.

**Comment by tecosaur** [comment]
Having just read the [Caddy v2.6 release notes](https://github.com/caddyserver/caddy/releases/tag/v2.6.0), it sounds like *Virtual file systems* could be useful for serving local git repos transparently.
